systems/core.py

```diff
@@ -128,6 +128,36 @@ def __call__(self) -> None:
         return self.run()
 
 
+class BrowserProxy:
+
+    def __init__(self, core_frontend_address: str, websocket_address: str):
+        """Transport bridge proxy for communication with broswer.
+
+        Args:
+            core_frontend_address (str): Address of frontend connection
+            websocket_address (str): Address of websocket binding
+        """
+        context = zmq.Context.instance()
+
+        self.identity = u'browser'
+
+        self.core_frontend_address = core_frontend_address
+        self.websocket_address = websocket_address
+
+        self.browser_socket = context.socket(zmq.DEALER)
+        self.browser_socket.identity = self.identity.encode('ascii')
+        self.core_socket = context.socket(zmq.DEALER)
+        self.core_socket.identity = self.identity.encode('ascii')
+
+    def run(self):
```

Comment:
Is there code here to receive data from UI and push to Core for handling?

Reply:
yeerrrr that's what zmq.proxy is doing